tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md

```diff
@@ -41,7 +41,7 @@ From the products service that you created in the previous tutorial, we just wan
 
     ```Shell/Bash
     mvn -B archetype:generate -DarchetypeArtifactId=cds-services-archetype -DarchetypeGroupId=com.sap.cds \
-    -DarchetypeVersion=RELEASE \
+    -DarchetypeVersion=RELEASE -DjdkVersion=11 \
```

Comment:
Align with https://github.com/eugene-andreev/Tutorials/blob/03faa7617750eb91ef5dc31f9417620331b04bb2/tutorials/cp-cap-java-app-studio/cp-cap-java-app-studio.md?plain=1#L87
Otherwise the project doesn't build

Reply:
Good catch! Thank you :)